api/v1/model/src/commonMain/kotlin/PackageCurationData.kt

```diff
@@ -38,4 +38,6 @@ data class PackageCurationData(
     val declaredLicenseMapping: Map<String, String>,
     val sourceCodeOrigins: List<SourceCodeOrigin>?,
     val labels: Map<String, String>,
+    /** The name of the curation provider this curation came from. */
```

Comment:
Nit: As the property's name is quite telling already, and we do not have comments for the other properties either, I think we should just omit this comment.